src/Libraries/CoreNodes/Color.cs

```diff
@@ -269,7 +279,12 @@ public static Color Multiply(Color c1, double div)
                 (int)(c1.Green * div),
                 (int)(c1.Blue * div));
         }
-
+        /// <summary>
+        /// Returns a new color after dividing an input color's ARGB values with an input divisor.
```

Comment:
Change to 'Divide an input color with a number divider to produce a brighter color and remove color tint.'